docs/_ext/single_sourced_data.py

```diff
@@ -6,26 +6,30 @@
 from copy import copy
 from pathlib import Path
 from re import match
-from typing import Any, Dict, List, Tuple, Union
+from typing import Any
```

Comment:
This file should be excluded from linting because it is a copy from external repositories and it's easier to keep such files in-sync across the board when they don't produce severe merge conflicts.

Reply:
Scratch that. I confused it with another file.